src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -145,7 +145,7 @@ function useChatUtilsHook(): UseChatUtilsHook {
   const goToMoveChat = (channel: ChannelList) => {
     setChat({
       ...chat,
-      selectedChannel: channel
+      selectedChannel: channel as unknown as null
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the change here is casting `channel` to `unknown` then to `null`. This doesn't seem correct because it will always set `selectedChannel` to `null` regardless of the `channel` value. It should be reverted back to the original code where `selectedChannel` is set to `channel`.
```diff
-      selectedChannel: channel as unknown as null
+      selectedChannel: channel
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->